apps/server/src/shared/common/loggable/referenced-entity-not-found-loggable.ts

```diff
@@ -0,0 +1,21 @@
+import { ErrorLogMessage, Loggable, LogMessage, ValidationErrorLogMessage } from '@src/core/logger';
+import { EntityId } from '../../domain/types';
+
+export class ReferencedEntityNotFoundLoggable implements Loggable {
+	constructor(
+		private readonly referencedId: EntityId,
+		private readonly sourceEntityName: string,
```

Comment:
this is a common loggable so it should be more flexible and have the same constructor order as existing loggables e.g. NotFoundLoggableException. Also using a different name in the constructor as in the log massage might be confusing.

You would have to declare the entity name and id as well as the referenced entity name (or the field name on the source entity) and id.